andela/mrm_api
# #159888760 edit list users query

**What does this PR do?**

Adds a user's name and profile photo URL to the `create user` mutation
Ensure a user list query returns a user's information, with name and photo included
A user's access level is also included in the query

**How should this be manually tested?**

Ensure the database is up to date by running **`alembic upgrade head`**
Run the server and execute the query as shown in the image below:

<img width="1033" alt="screen shot 2018-08-23 at 18 11 06" src="https://user-images.githubusercontent.com/26790578/44534839-5bb20780-a701-11e8-90c2-a6970256a5d5.png">

**What are relevant PivotalTracker stories?**

ID: #159888760 
Strory: Edit list users query
